renderlib/FileReaderZarr.cpp

```diff
@@ -7,13 +7,42 @@
 
 #include "tensorstore/context.h"
 #include "tensorstore/index_space/dim_expression.h"
+#include "tensorstore/kvstore/generation.h"
 #include "tensorstore/open.h"
 
 #include <algorithm>
 #include <chrono>
 #include <map>
 #include <set>
 
+static bool
+isCloud(const std::string& filepath)
+{
+  return filepath.find("http") == 0;
+}
+
+static nlohmann::json
+getKvStoreDriverParams(const std::string& filepath, const std::string& subpath)
+{
+  if (isCloud(filepath)) {
+    return { { "driver", "http" }, { "base_url", filepath }, { "path", subpath } };
+  } else {
+    // if file path does not end with slash then add one
+    // TODO maybe use std::filesystem::path for cross-platform?
```

Comment:
the slight ugliness that follows was done for expediency and it needs to be made more robust.  Expediency: so that it can be built and distributed internally for zarr testing.